website/data/en/vendors/consulting.toml

```diff
@@ -15,13 +15,13 @@ name = "A-Team Systems"
 url = "http://www.ateamsystems.com/"
 description = "A-Team Systems has over 15 years of experience helping clients leverage the power of in combination with other open source software such as PHP, MySQL and Apache in a secure, customized and cohesive way. We have been with since version 1.0! We can help you with your existing deployment or future plans: From encrypted backups to web and mail servers. Visit our <a href=\"http://www.ateamsystems.com/\">website</a> for more information."
 
-[[consulting]]
-id = "anonbsd"
-category = "samerica"
-name = "AnonBSD Inc."
-url = "https://sites.google.com/site/anonbsd/"
-description = "We provide IT environment consulting. Support in FreeBSD, OpenBSD, Linux (Debian, RedHat, SUSE, Gentoo, Slackware) and OpenSolaris. High availability, virtualization, monitoring, security, deployment, network infrastructure. We service in São Paulo/SP and Campinas/SP in Brazil. Contact us via e-mail (<a href=\"mailto:ricardo.ichizo@gmail.com\">ricardo.ichizo@gmail.com</a>) or by phone at +55 (19) 8202-1567."
-
+# [[consulting]]
+# id = "anonbsd"
+# category = "samerica"
+# name = "AnonBSD Inc."
+# url = "https://sites.google.com/site/anonbsd/"
+# description = "We provide IT environment consulting. Support in FreeBSD, OpenBSD, Linux (Debian, RedHat, SUSE, Gentoo, Slackware) and OpenSolaris. High availability, virtualization, monitoring, security, deployment, network infrastructure. We service in São Paulo/SP and Campinas/SP in Brazil. Contact us via e-mail (<a href=\"mailto:ricardo.ichizo@gmail.com\">ricardo.ichizo@gmail.com</a>) or by phone at +55 (19) 8202-1567."
+# 
```

Comment:
<https://web.archive.org/web/20101223001810/https://sites.google.com/site/anonbsd/> (2010) ▶ <https://web.archive.org/web/20201017085849/https://sites.google.com/site/anonbsd/contact> (2020) …

Reply:
Ricardo Ichizo sends regards, and requests removal; 

> … we do not do that kind of work anymore.